### DISCUSSION TITLE
Extract information about updated packages from poetry.lock file

### DISCUSSION THREAD

**Discussion by heckad** [renovatebot/renovate#24963] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.3.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

For example, we have `pyproject.toml` with 2 dependencies.
```
[tool.poetry.group.base.dependencies]
SQLAlchemy = "^1.3.0"
gino = "^1.0.1"
```

the first rule says that sqlalchemy can be `>=1.3.0 && < 2.0.0` and renovate create merge request with updating to `1.4.49` but gino has a constraint which says that sqlalchemy should be `<1.4.0`. In the debug log renovate call the `poetry lock ...` command which solves such mistakes and places the correct version, but the merge request message contains information about updating sqlalchemy to `1.4.49`. It seems that for the description it is necessary to pull out the changes from the lock file.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by ewjoachim** [comment]
I have the same issue (I think it's the same) here https://github.com/procrastinate-org/procrastinate/pulls?q=is%3Apr+sort%3Aupdated-desc+7.2.6+ where renovate has claimed a handful of times to update sphinx, but it actually didn't do so.
